src/main/java/com/solvd/webtesting/page/HomePage.java

```diff
@@ -0,0 +1,90 @@
+package com.solvd.webtesting.page;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import com.solvd.webtesting.elements.ProductMenuBar;
+import com.solvd.webtesting.elements.SearchSection;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends AbstractPage {
+
+    @FindBy(xpath = "//*[contains(@class,'styles_primaryButton__30pEq')]")
+    private ExtendedWebElement cookieButton;
+
+    @FindBy(xpath = "//*[contains(@class,'styles_userToolsToggler__imcSl')]")
+    private ExtendedWebElement accountMenu;
+
+    @FindBy(css = ".userToolsBtn")
+    private ExtendedWebElement loginButton;
+
+    @FindBy(css = ".styles_userTools__2J7cp.undefined")
+    private ExtendedWebElement accountMenuAfterLogin;
+
+    @FindBy(css = ".userToolsSubtitle")
+    private ExtendedWebElement userToolsTitle;
+
+    @FindBy(xpath =  "//*[contains(@class,'styles_headerReactWrapper__3U15a') and contains(@class,'styles_headerReactWrapperAlt__3bM5y')]")
+    private com.solvd.webtesting.elements.ProductMenuBar ProductMenuBar;
+
+    @FindBy(xpath = "//*[contains(@class,'styles_headerReactLine__IWu_8') and contains(@class,'styles_headerReactLineSecond__tp1U0')]")
+    private com.solvd.webtesting.elements.SearchSection SearchSection;
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        setPageURL("");
+    }
```

Comment:
Why did you set empty URL?

Reply:
I saw in carina demo URL set as : 
        setPageAbsoluteURL(R.CONFIG.get(Configuration.Parameter.URL.getKey()));
      But during Laba we used to environment configurations: 
      env=PROD
      PROD.base=https://21vek.by/
       and if I understand correctly that carina pulls up the base url or am I mistaken?